### ISSUE TITLE
Fix FBX import errors

### ISSUE THREAD

**Issue by repwolfe** [repwolfe/ProjectMishkan#10]
Fix the following errors that appear when importing FBX files exported from Blender:

> mizbeach_zahav has degenerate tangent bases which will result in incorrect shading.
> 
> Warning: No smoothing group information was found in this FBX scene. Please make sure to enable "Export Smoothing Group"

**Comment by repwolfe** [comment]
This is related to #1

**Comment by haimhi** [comment]
i'm working on this. seems to have to do with the export plugin shipped with blender.

**Comment by haimhi** [comment]
i think i got it fixed. Ber's newer models (aron etc.) and a new export plugin do the trick.

**Comment by repwolfe** [comment]
This seems fixed

**Comment by haimhi** [comment]
not quite. see the menorah and the shulchan for example. need to consult
with and/or get Dov Ber back on the models.

On Mon, Jun 29, 2015 at 2:02 PM, Robbie Wolfe notifications@github.com
wrote:

> Closed #10 https://github.com/repwolfe/ProjectMishkan/issues/10.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/repwolfe/ProjectMishkan/issues/10#event-342874208.

## 

בס"ד

**Comment by repwolfe** [comment]
Okay then reopening.
